apps/web/components/ui/CartProductCard/CartProductCard.vue

```diff
@@ -4,63 +4,86 @@
     data-testid="cart-product-card"
   >
     <div class="relative overflow-hidden rounded-md w-[100px] sm:w-[176px]">
-      <SfLink :tag="NuxtLink" :to="`${paths.product}${slug}`">
+      <SfLink
+        :tag="NuxtLink"
+        :to="`${paths.product}${cartGetters.getItemName(cartItem)}-${cartGetters.getVariationId(cartItem)}`"
+      >
         <NuxtImg
           class="w-full h-auto border rounded-md border-neutral-200"
-          :src="imageUrl ?? '/images/product.webp'"
-          :alt="imageAlt ?? ''"
+          :src="cartGetters.getItemImage(cartItem) || '/images/product.webp'"
```

Comment:
**question**: Why do we need to keep the fallback?

Reply:
@ksted 

We need the fallback because the image is lazy-loaded into the page. We don't have the image yet when the component is rendered. This currently does not have any UI effect on our app because of the fast image load time, but for a bigger image we would need a default until image loads. This currently does however have a code effect, the component not accepting empty string in the **src** attribute.

I placed a todo so we know to eventually find a proper default image.